### ISSUE TITLE
Announce "loading" in accessibility only when progress has not finished

### ISSUE THREAD

**Issue by rocketsroger** [mozilla-mobile/android-components#10241]
Currently we announce loading to accessibility when progress is at 100%.  This is caused by short loading that goes straight to 100%.  Visibility this is not an issue because the progress bar is not visible at 100%.  However, for Talkback users this is very noticeable when they keep hearing "loading" between paragraphs/lines.

Stop announcing "loading" to accessibility when progress is at 100%.
This is for https://github.com/mozilla-mobile/fenix/issues/17080
